subtests/docker_cli/stop/stop.py

```diff
@@ -77,32 +77,24 @@ def run_once(self):
 
     def check_output(self):
         """
-        Check that config[check_stdout] is present in the execute docker stdout
+        Check that config[check_stdout] is present in the executed docker
+        command's stdout; or, if 'check_output_inverted' config is set,
+        check that it is absent.
```

Comment:
OMG that's confusing, why not just have another configuration option with a different name.  Who wrote this? They should be shot :grinning: